onnx/backend/test/cpp/test_main.cc

```diff
@@ -2,11 +2,12 @@
 #include "gtest_utils.h"
 
 GTEST_API_ int main(int argc, char** argv) {
-  if (argc != 2) {
+  if (argc < 2) {
     std::cerr << "target directory must be given!" << std::endl;
     return EXIT_FAILURE;
   }
-  std::string target_dir = argv[1];
+  argc--;
```

Comment:
Sorry for late comment.

But it's not clear why this `argc--` magic happens here, can you add a comment? (and that we always expect target directory to be the last param).

Reply:
oops, sure we need to add comments here, let me do it.